### DISCUSSION TITLE
Windows 11 - no keyboard & no mouse connected - Rustdesk issue

### DISCUSSION THREAD

**Discussion by malobik** [rustdesk/rustdesk#4347] [Q&A]
Hello,
I am looking for advice/help. 
I have computer with Windows 11 that I can nicely connect to via RustDesk and all works just fine. However once I physically disconnect from that computer keybord, mouse and monitor, problems start to occure. It gets very slow, windows do not want to open, start pannel on windows does not pop up etc. Once I connect back these devices it again works like charm. It seems to me like windows 11 is kind of looking to have them connected in order to work properly. Any idea what needs to be adjusted changed in Windows 11 to overcome this issues? I simply want that computer to be only connected to power and network and accesible via remote desktop only.
Thank you

**Comment by rustdesk** [comment]
frankly, we do not know either
